src/lib.rs

```diff
@@ -4,14 +4,22 @@
 
 #![forbid(unsafe_code)]
 
+/// Re-exporting `wasm-bindgen-test`.
+#[cfg(target_arch = "wasm32")]
+#[doc(hidden)]
+pub use wasm_bindgen_test;
```

Comment:
I'd prefer this be exported in the `__private` module.

Reply:
Ah, indeed, that makes sense.
Done.